plutus-tx-plugin/test/Budget/9.6/show.pir.golden

```diff
@@ -324,16 +318,13 @@ let
   data Bool | Bool_match where
     True : Bool
     False : Bool
-  !a : integer = trace {integer} (`$fShowInteger_$cshow` x) x
+  !a : integer
+    = trace {integer} (`$fShowInteger_$cshow` -1234567890) -1234567890
   !c : integer
     = trace
         {integer}
         (concatBuiltinStrings
-           ((let
-                !len : integer = lengthOfByteString s
-              in
-              go (`$fEnumBool_$cenumFromTo` 0 (subtractInteger len 1)))
-              (Nil {string})))
+           (go (`$fEnumBool_$cenumFromTo` 0 17) (Nil {string})))
```

Comment:
This seems right, can we assume that the (huge) `uplc` version below is fine as well? :smile: